src/components/attendee/SearchOverlay/SearchOverlay.tsx

```diff
@@ -66,6 +73,19 @@ export const SearchOverlay: React.FC<SearchOverlayProps> = ({ onClose }) => {
     [worldSpaces]
   );
 
+  const { liveAndFutureEvents } = useVenueScheduleEvents();
+
+  const foundEvents = useMemo<ScheduledEvent[]>(() => {
+    if (!searchQuery) return ALWAYS_EMPTY_ARRAY;
+
+    /* @debt we really shouldn't be using the index as part of the key here, it's unstable.. but portals don't have a unique identifier */
```

Comment:
is this comment still relevant? or maybe we should move it?

Reply:
removed [09146a2](https://github.com/sparkletown/sparkle/pull/3074/commits/09146a2a8a1fd2ccbb351fd1056f0487e6181767)